app/models/user.rb

```diff
@@ -202,10 +202,6 @@ def completed_containers_with_lookahead(sequence, organization, lookahead: 1)
     sequence[0..count + lookahead - 1]
   end
 
-  # Tells if the given user can discuss in an organization
-  #
-  # This is true only when this organization has the forum enabled and the user
-  # has the discusser pseudo-permission and the discusser is trusted
```

Comment:
I don't think this added much and was outdated anyway, because it now depends on the organization's access mode - and then even on the default mode there's actually more conditions for letting discuss, such as the user not being banned and not being in an exam (which I just added). 

So I removed the comment.